zookeeper-server/src/main/java/org/apache/zookeeper/metrics/MetricsProviderLifeCycleException.java

```diff
@@ -19,7 +19,7 @@
 package org.apache.zookeeper.metrics;
 
 /**
- * A generic exception thrown during the licecycle of a MetricsProvider.
+ * A generic exception thrown during the life cycle of a MetricsProvider.
```

Comment:
```suggestion
 * A generic exception thrown during the lifecycle of a MetricsProvider.
```

I think both are correct, but I saw multiple usages of "lifecycle" but no "life cycle".

Reply:
Thank you, maybe I haven't seen much, you're right